wallet/chainntfns.go

```diff
@@ -293,6 +294,15 @@ func (w *Wallet) onBlockConnected(serializedBlockHeader []byte, transactions [][
 	w.NtfnServer.notifyMainChainTipChanged(chainTipChanges)
 	w.NtfnServer.sendAttachedBlockNotification()
 
+	// Parse vote version from extended bits.  Note that once ExtendedBits
+	// are used for other information as well this will need to be updated
+	// to accomodate.
+	voteVersion := binary.LittleEndian.Uint32(w.VoteBits().ExtendedBits)
+	if voteVersion < blockHeader.StakeVersion {
+		log.Warnf("Old vote version detected (v%v), please update your "+
+			"wallet to the latest version.", voteVersion)
```

Comment:
ooh, correct, I misread because this was parsing the bits out of the slice.

A better way to do this is to call `voteVersion(w.chainParams)` which will just return the vote version that the wallet writes to the slice, instead of needing to read it out of the slice.